requirements/requirements_ansible.txt

```diff
@@ -105,14 +110,16 @@ requests==2.22.0
 requestsexceptions==1.4.0  # via openstacksdk
 rsa==4.0                  # via google-auth
 s3transfer==0.2.1         # via boto3
+scandir==1.10.0; python_version < "3" # via pathlib2
 selectors2==2.0.1         # via ncclient
-six==1.13.0               # via azure-cli-core, bcrypt, cryptography, google-auth, isodate, keystoneauth1, knack, munch, ncclient, openstacksdk, ovirt-engine-sdk-python, packaging, pynacl, pyopenssl, python-dateutil, pyvmomi, pywinrm, requests-credssp, stevedore
+six==1.13.0; python_version < "3" # via azure-cli-core, bcrypt, cryptography, google-auth, isodate, keystoneauth1, knack, more-itertools, munch, ncclient, openstacksdk, ovirt-engine-sdk-python, packaging, pathlib2, pynacl, pyopenssl, python-dateutil, pyvmomi, pywinrm, requests-credssp, stevedore
 stevedore==1.31.0         # via keystoneauth1
 tabulate==0.8.2           # via azure-cli-core, knack
 typing==3.7.4.1; python_version < "3" # via msrest
 urllib3==1.25.7           # via botocore, requests
 wheel==0.30.0             # via azure-cli-core
 xmltodict==0.12.0         # via pywinrm
+zipp==1.0.0               # via importlib-metadata
```

Comment:
Once I got testing running, this led to a snag, see:

https://github.com/jaraco/zipp/blob/0ecb3a3953e09a2e6b4e1a91b4d6b5ff88a303c7/pyproject.toml#L2

```
        Looking in links: tower/requirements/vendor
        ERROR: Could not find a version that satisfies the requirement setuptools>=42 (from versions: 41.6.0)
        ERROR: No matching distribution found for setuptools>=42
```

By requiring setuptools version 42 or higher, this is creating a butterfly effect of consequences.

Since the Azure dependencies are _not_ being upgraded, we will probably pin `argcomplete` with some notes as to why. That additional dep chain was brought about by https://github.com/kislyuk/argcomplete/pull/283 (not surprisingly), with introduction version of `v1.11.0`. That leaves the last argcomplete version before this (incompatible) upgrade to 1.10.3, which is what we already had